src/wrap.js

```diff
@@ -14,7 +14,11 @@ jQuery.fn.extend( {
 			}
 
 			// The elements to wrap the target around
-			wrap = jQuery( html, this[ 0 ].ownerDocument ).eq( 0 ).clone( true );
+			wrapper = jQuery( html, this[ 0 ].ownerDocument );
```

Comment:
This doesn’t work at all as there’s no such variable, it will just crash. Did you run this code locally?